style.css

```diff
@@ -87,3 +88,278 @@ body {
   justify-content: flex-start;
   padding-left: 5px;
 }
+
+.page1 {
+  background-color: var(--v3);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0%;
+  justify-content: center;
+  background-repeat: no-repeat;
+}
+
+.tituloMR {
+  font-style: normal;
+  font-size: 40px;
+  color: var(--v2);
+  font-family: 'Crete Round', serif;
+  padding-left: 15px;
+  margin-bottom: 0%;
+  padding-top: 802px;
+}
+
+.vector {
+  border: 1px solid #212121;
+  color: var(--v2);
+  left: 24px;
+  right: 24px;
+}
+
+.yoga {
+  left: 24px;
+  padding-top: 104px;
+  padding-left: 15px;
+  padding-right: 30px;
+}
+
+.multi {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  padding-left: 15px;
+  padding-right: 30px;
+}
+
+.parrafo1 {
+  font-family: 'Inter', sans-serif;
+  padding-left: 15px;
+  padding-right: 30 px;
+  font-size: 16px;
+  color: #091e42;
+}
+
+.list_programs {
+  display: flex;
+  align-content: space-between;
+  flex-direction: row;
+  padding: 10px 12px;
+}
+
+.boton1 {
+  height: 48px;
+  background: #ff6b00;
+  margin-left: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(./images/fondo.svg);
+  background-repeat: no-repeat;
+  width: 327px;
```

Comment:
- [ ] Please check your design on the projects class; the ` background-repeat` and `width` property affects the design. [Check this for reference ](https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat)